### DISCUSSION TITLE
What does the star feature for a repo do?

### DISCUSSION THREAD

**Discussion by siliconvolley** [community/community#63174] [Discussions]
### Select Topic Area

Question

### Body

Star a repo

**Comment by shettyvarshaa** [comment]
A feature that is similar to liked videos in youtube or saved feature for posts in social media

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
